app/middleware/csp.js

```diff
@@ -1,56 +1,58 @@
 const helmet = require('helmet')
+const Sentry = require('../utils/sentry').initialiseSentry()
+const paths = require('../paths')
+const express = require('express')
+const { rateLimit } = require('express-rate-limit')
+const logger = require('../utils/logger')(__filename)
+const hasSubstr = require('../utils/has-substr')
 
 const sendCspHeader = process.env.CSP_SEND_HEADER === 'true'
 const enforceCsp = process.env.CSP_ENFORCE === 'true'
-const cspReportUri = process.env.CSP_REPORT_URI
-const environment = process.env.ENVIRONMENT
 const allowUnsafeEvalScripts = process.env.CSP_ALLOW_UNSAFE_EVAL_SCRIPTS === 'true'
 
-const sentryCspReportUri = `${cspReportUri}&sentry_environment=${environment}`
```

Comment:
Do we no longer need to specify the environment to Sentry? I can't quite see where we're doing it now

Reply:
we send via the DSN now using the Sentry Node client